source/index.html.md

```diff
@@ -598,8 +598,9 @@ Exactly one payment method must be used.
     This is regardless of whether a stored payment credential is being used.
     <br />
     For compliance reasons there should be a previous approved transaction (for
-    the cardholder and the merchant) marked with <code>storing</code> before
-    <code>initiator</code> may be <code>merchant</code>.
+    the combination of cardholder and merchant) where
```

Comment:
```suggestion
    the combination of card and merchant) where
```

Strictly speaking, it's the card, not the cardholder, because a cardholder might have more than a single card :man_shrugging: Perhaps we could or should write PAN instead, but I leans towards just card.

Other than that I think the clarification makes sense and is clear.